releasenotes/notes/0.46/prepare-0.46.0-c3857463a3c052f7.yaml

```diff
@@ -0,0 +1,620 @@
+---
+prelude: >
+    The 0.46.0 release is the final minor version release for the 0.x series.
+    This release primarily adds new deprecation warnings for API changes coming
+    in the future major version release 1.0.0. It is fully compatible with the
+    Qiskit 0.45.x releases. It is strongly recommended that you upgrade from
+    0.45.x to 0.46.0 so that you're able to see the warnings about which
+    interfaces will change with Qiskit 1.0.0.
+
+    The 0.46.x release series will continue to be supported and recieve
+    bugfix and security fixes via patch releases for 6 months after this
+    release. For more details on the release schedule and support cycle
+    see: https://docs.quantum.ibm.com/start/install#qiskit-versioning
+    which documents the release schedule in more detail.
+
+    .. note::
+
+      If your project depends on Qiskit, it may rely on functionality that will no longer
+      be supported in Qiskit 1.0.0. For this reason, we recommend that you proactively cap your
+      supported version to ``<1.0``. Qiskit 1.0.0 is scheduled to release approximately two weeks
+      after Qiskit 0.46.0, on the 15th of February 2024, and might not yet be available when
```

Comment:
I forgot to leave a comment on the date in 0.45.3 (where this was copied from) I should use isoformat here as February doesn't translate well for everyone.